# Internal patch.
# Change the output format of the test harness so we can handle
# it better with our master test results file.
# Not for upstream.
--- test/common/tests.c	2013-07-31 06:50:34.000000000 -0700
+++ test/common/tests.c	2015-03-17 19:13:48.294707531 -0700
@@ -154,7 +154,7 @@
 static void print_prefix(int n)
 {
     if (quiet) {
-        printf("\r%s%.*s %2u/%2u ", test_suite, 
+        printf("\n%s%.*s %2u/%2u ", test_suite, 
                (int) (strlen(dots) - strlen(test_suite)), dots,
                n + 1, count);
     }
@@ -163,7 +163,7 @@
 	    printf("    %s ", dots);
         }
         else {
-            printf("\r%2d. %s%.*s ", n, test_name, 
+            printf("\n%2d. %s%.*s ", n, test_name, 
                (int) (strlen(dots) - strlen(test_name)), dots);
         }
     }
@@ -393,7 +393,7 @@
             printf("<- all tests skipped for `%s'.\n", test_suite);
     } else {
         if (quiet) {
-            printf("\r%s%.*s %2u/%2u ", test_suite, 
+            printf("\n%s%.*s %2u/%2u ", test_suite, 
                    (int) (strlen(dots) - strlen(test_suite)), dots,
                    passes, count);
             if (fails == 0) {
